app/waf/src/controllers/endpointpolicy.controller.ts

```diff
@@ -23,10 +23,17 @@ import {
   ServiceEndpointpolicyAssociationRepository,
 } from '../repositories';
 import {Schema, Response, CollectionResponse} from '.';
-import uuid = require('uuid');
 
 const prefix = '/adcaas/v1';
 
+const createDesc: string = 'Endpointpolicy resource that need to be created';
```

Comment:
These statements are used only once, declare them while using should be better. Needn't declare them separately.

Reply:
ok for me.